mapstruct-unmapped-policy/src/test/java/com/baeldung/mapper/DocumentMapperUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.mapper;
+
+import com.baeldung.dto.DocumentDTO;
+import com.baeldung.entity.Document;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+
+class DocumentMapperUnitTest {
+
+    @Test
+    void givenDocumentEntityToDocumentDto_whenMaps_thenCorrect() {
+        Document entity = new Document();
+        entity.setId(1);
+        entity.setTitle("Price 13-42");
+        entity.setText("List of positions.......");
+        entity.setModificationTime(new Date());
+
+        DocumentDTO dto = DocumentMapper.INSTANCE.documentToDocumentDTO(entity);
+
+        assertEquals(dto.getId(), entity.getId());
+        assertEquals(dto.getTitle(), entity.getTitle());
+        assertEquals(dto.getText(), entity.getText());
```

Comment:
I find it inconsistent to use JUnit 4 assertions with JUnit 5. Please can you switch to the `jupiter` assertions, or even try assertJ, which is much nicer - `assertThat(dto.getId()).isEqualTo(entity.getId());`